school_attendance/models/school_attendance.py

```diff
@@ -0,0 +1,1204 @@
+            percentage = 0.0
+            if attendance_sheet_data.one:
+                att_count = att_count + 1
+            if attendance_sheet_data.two:
+                att_count = att_count + 1
+            if attendance_sheet_data.three:
+                att_count = att_count + 1
+            if attendance_sheet_data.four:
+                att_count = att_count + 1
+            if attendance_sheet_data.five:
+                att_count = att_count + 1
+            if attendance_sheet_data.six:
+                att_count = att_count + 1
+            if attendance_sheet_data.seven:
+                att_count = att_count + 1
+            if attendance_sheet_data.eight:
+                att_count = att_count + 1
+            if attendance_sheet_data.nine:
+                att_count = att_count + 1
+            if attendance_sheet_data.ten:
+                att_count = att_count + 1
+
+            if attendance_sheet_data.one_1:
+                att_count = att_count + 1
+            if attendance_sheet_data.one_2:
+                att_count = att_count + 1
+            if attendance_sheet_data.one_3:
+                att_count = att_count + 1
+            if attendance_sheet_data.one_4:
+                att_count = att_count + 1
+            if attendance_sheet_data.one_5:
+                att_count = att_count + 1
+            if attendance_sheet_data.one_6:
+                att_count = att_count + 1
+            if attendance_sheet_data.one_7:
+                att_count = att_count + 1
+            if attendance_sheet_data.one_8:
+                att_count = att_count + 1
+            if attendance_sheet_data.one_9:
+                att_count = att_count + 1
+            if attendance_sheet_data.one_0:
+                att_count = att_count + 1
+
+            if attendance_sheet_data.two_1:
+                att_count = att_count + 1
+            if attendance_sheet_data.two_2:
+                att_count = att_count + 1
+            if attendance_sheet_data.two_3:
+                att_count = att_count + 1
+            if attendance_sheet_data.two_4:
+                att_count = att_count + 1
+            if attendance_sheet_data.two_5:
+                att_count = att_count + 1
+            if attendance_sheet_data.two_6:
+                att_count = att_count + 1
+            if attendance_sheet_data.two_7:
+                att_count = att_count + 1
+            if attendance_sheet_data.two_8:
+                att_count = att_count + 1
+            if attendance_sheet_data.two_9:
+                att_count = att_count + 1
+            if attendance_sheet_data.two_0:
+                att_count = att_count + 1
+            if attendance_sheet_data.three_1:
+                att_count = att_count + 1
+            percentage = (float(att_count / 31.00)) * 100
+            attendance_sheet_data.percentage = percentage
+        return res
+
+    _description = "Attendance Sheet Line"
+    _name = "attendance.sheet.line"
+    _order = "roll_no"
+
```

Comment:
We should utilise HR modules smartly and revamp. @AnuKPatel I am sure you should not miss the basic idea.